lib/pure/net.nim

```diff
@@ -1444,8 +1444,6 @@ proc recv*(socket: Socket, data: var string, size: int, timeout = -1,
   ## A timeout may be specified in milliseconds, if enough data is not received
   ## within the time specified a TimeoutError exception will be raised.
   ##
-  ## **Note**: `data` must be initialised.
```

Comment:
this note is from 7 years ago - now strings are initialized by default, also this procedure resizes the string by itself.